### PR TITLE
Fix link to the proper version in welcome dialog

### LINKED ISSUE #15412: Incorrect version is displayed in the welcome window
### Summary
Incorrect version is displayed in the welcome window

### Step to reproduce
Install a clean version of MODX 3.x

### Observed behavior
photo_2021-02-24_03-06-21

### Expected behavior
Should display the correct version of MODX Revolution 3.x

https://github.com/modxcms/revolution/blob/3.x/_build/data/transport.core.system_settings.php#L1835

### Related issue(s)/PR(s)
NA

### Environment
MODX version: Latest version with Github

### PR DESCRIPTION
**Pull request by Ibochkarev**
### What does it do?
Fix incorrect version is displayed in the welcome window

### Why is it needed?
Fix path for preview the page in system settings

### How to test
Install clean build

### Related issue(s)/PR(s)
Resolves #15412

### CODE REVIEW

**Review by alroniks — APPROVED** [review]

**Review by sergant210 — APPROVED** [review]

**Review by theboxer — APPROVED** [review]

**Comment by alroniks** [comment]
@Ibochkarev What about the upgrading script?

**Comment by Ibochkarev** [comment]
> @Ibochkarev What about the upgrading script?

if you need it, I will do it without problems

File: setup/includes/upgrades/common/3.0.0-update-sys-setting_welcome_screen_url.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Update the URL welcome_screen_url
+ */
+use MODX\Revolution\modSystemSetting;
+
+/** @var modSystemSetting $welcome_screen_url */
+$welcome_screen_url = $modx->getObject(modSystemSetting::class, [
```
Comment (alroniks) [inline setup/includes/upgrades/common/3.0.0-update-sys-setting_welcome_screen_url.php:8]: Please, use PSR-12 for the new code.

**Review by alroniks — CHANGES_REQUESTED** [review]

File: setup/includes/upgrades/common/3.0.0-update-sys-setting_welcome_screen_url.php (hunk 1 above)
Comment (Ibochkarev) [reply setup/includes/upgrades/common/3.0.0-update-sys-setting_welcome_screen_url.php:8]: Done

**Review by alroniks — APPROVED** [review]

**Pull request merged**